cairo/ethereum/cancun/fork_types.cairo

```diff
@@ -83,6 +83,21 @@ struct MappingAddressAccount {
     value: MappingAddressAccountStruct*,
 }
 
+struct BytesU256DictAccess {
+    key: Bytes,
+    prev_value: U256,
+    new_value: U256,
+}
+
+struct MappingBytesU256Struct {
+    dict_ptr_start: BytesU256DictAccess*,
+    dict_ptr: BytesU256DictAccess*,
+}
```

Comment:
technically this is not wrong, but I see that this is used for the storage trie where the keys are badly typed in the EELS (should be bytes32 and not bytes)

Reply:
I prefer not to touch this for now to avoid disrepancies